07_errors/jimbotech/sync_mapstore.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"math/rand"
+	"sync"
+)
+
+// SyncMapStore stores puppies threadsafe.
+type SyncMapStore struct {
```

Comment:
the filename should be `syncmapstore.go`. But I would call the type `SyncStore` because shorter is better. From the outside, no-one cares that it is a map - it's a store. So don't name it with "Map". Which means the filename could be `syncstore.go`

Reply:
Done